tests/test.py

```diff
@@ -50,13 +50,34 @@ def test_search_uploads(self):
 
     def test_get_artist(self):
         results = youtube.get_artist("MPLAUCmMUZbaYdNH0bEd1PAlAqsA")
-        self.assertEqual(len(results), 13)
+        self.assertEqual(len(results), 14)
+
+        # test correctness of related artists
+        related = results['related']['results']
+
+        # all should have 3 keys per item
+        self.assertEqual(
+            len([x for x in related if set(x.keys()) == {"browseId", "subscribers", "title"}]),
+            len(related))
+
+        # test names of some related artists
```

Comment:
I'd advise against testing artist names specifically since the list may change for unrelated reasons (different recommendations), causing the test to break